docs/rules/0122/embedded-resource.md

```diff
@@ -0,0 +1,107 @@
+---
+rule:
+  aip: 122
+  name: [core, '0122', embedded-resource]
+  summary: Resource references should not be embedded resources.
+permalink: /122/embedded-resource
+redirect_from:
+  - /0122/embedded-resource
+---
+
+# Resource reference type
+
+This rule enforces that references to resource are via
+`google.api.resource_reference`, not by embedding the resource message, as
+mandated in [AIP-122][].
```

Comment:
is it worth mentioning there are exceptions? just thinking about the corpeng folks who like this feature running into this all the time.

we could just disable the rule in the corpeng directory too.

Reply:
I don't think we need to enumerate exceptions here since the AIP will do so, that's what the "as mandated" is for. Probably best not to duplicate every edge case to guidance in the linter documentation.

I will make sure corpeng get a heads up to disable this rule before it gets into prod.